early-init.el

```diff
@@ -88,6 +88,16 @@
 (defvar rational-prefer-guix-packages (rational-using-guix-emacs-p)
   "If t, expect packages to be installed via Guix by default.")
 
+(defcustom rational-load-custom-file nil
```

Comment:
The point of #112 was to load the custom.el file, ie default to opt-in due to worflow related updates from org mode (see `org-agenda-file-to-front` which eventually calls `org-store-new-agenda-file-list` which then calls `customize-save-variable` to store the updated agenda file list in the `custom-set-variables` form. The same is true for `org-remove-file` (it updates the `custom-set-variables` form). Also there is a case for `.dir-locals.el` updates to be stored as accepted. So, we would prefer a `t` default setting here, and then allow the user to change that to `nil` in their config. Unfortunately, using the Customize UI in this case fails and therefore requires the user to use elisp to code change this configuration. In view of that, I'd recommend `defvar` instead.